src/methylation/pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[project]
+name = "ugbio_methylation"
+version = "1.3.5-0dev-70"
+description = "Ultima Genomics methylation bioinfo utils"
+authors = [
+    { name = "Gat Krieger", email = "gat.krieger@ultimagen.com" },
+    { name = "Avigail Moldovan", email = "avigail.moldovan@ultimagen.com" },
+]
+dependencies = [
+    "ugbio_core",
```

Comment:
remove core. Not used.
I guess this will require to add some import some that are in core and used here